projects/fusion-ui/components/chip-filter/v3/chip-filter.component.ts

```diff
@@ -0,0 +1,10 @@
+import {ChangeDetectionStrategy, Component, OnDestroy, OnInit} from '@angular/core';
+import {ChipFilterBaseComponent} from '@ironsource/fusion-ui/components/chip-filter/common/base';
+
+@Component({
+    selector: 'fusion-chip-filter',
+    templateUrl: './chip-filter.component.html',
+    styleUrls: ['./chip-filter.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ChipFilterComponent extends ChipFilterBaseComponent implements OnInit, OnDestroy {}
```

Comment:
you can remove `implements OnInit, OnDestroy` when you extend class that implement those classes